pkg/operator/resource/resourceapply/admissionregistration_test.go

```diff
@@ -562,6 +562,138 @@ func TestApplyValidatingAdmissionPolicyConfiguration(t *testing.T) {
 	}
 }
 
+func TestDeleteValidatingAdmissionPolicy(t *testing.T) {
+	defaultPolicy := &admissionregistrationv1.ValidatingAdmissionPolicy{}
+	defaultPolicy.SetName("test")
+	deleteEvent := "ValidatingAdmissionPolicyDeleted"
+
+	tests := []struct {
+		name           string
+		expectModified bool
+		existing       func() *admissionregistrationv1.ValidatingAdmissionPolicy
+		input          func() *admissionregistrationv1.ValidatingAdmissionPolicy
+		checkUpdated   func(*admissionregistrationv1.ValidatingAdmissionPolicy) error
+		expectedEvents []string
+	}{
+		{
+			name:           "Should delete webhook if it exists",
```

Comment:
ValidatingAdmissionPolicies are not webhooks (compare with ValidatingWebhookConfiguration). This applies to this whole file - `existingHooks` below, `updatedHook`, other test names.  I would personally use term `policy` as a shortcut.